recipes/shared-memory-dict/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "shared-memory-dict" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shared-memory-dict-{{ version }}.tar.gz
+  sha256: 8294257e1a2466afb5e19a878a8c446325b8913d57d6d9578ccba905564a974d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+    - poetry
+  run:
+    - python >=3.6,<4.0
+
+test:
+  imports:
+    - shared_memory_dict
+  commands:
+    - pip check
+  requires:
+    - pip
+
+outputs:
+  - name: shared-memory-dict
+    requirements:
+      host: &id001
+        - pip
+        - python >=3.6,<4.0
+        - poetry
+      run:
+        - python >=3.6,<4.0
+  - name: shared-memory-dict-aiocache
+    requirements:
+      host: *id001
+      run:
+        - shared-memory-dict =={{ version }}
+        - aiocache >=0.11.1,<0.12.0
+  - name: shared-memory-dict-all
+    requirements:
+      host: *id001
+      run:
+        - shared-memory-dict =={{ version }}
+        - aiocache >=0.11.1,<0.12.0
+        - django >=3.0.8,<4.0.0
+  - name: shared-memory-dict-django
+    requirements:
+      host: *id001
+      run:
+        - shared-memory-dict =={{ version }}
+        - django >=3.0.8,<4.0.0
```

Comment:
This section needs to be formatted in a different way. Please have a look at the mathplotlib feedstock as an example on how to do it: https://github.com/conda-forge/matplotlib-feedstock/blob/main/recipe/meta.yaml

Reply:
See [778a669](https://github.com/conda-forge/staged-recipes/pull/19855/commits/778a66946d8d34b23308b0f2008d9758f1981e48). FYI, I used the new grayskull feature to [automatically add optional dependencies](https://github.com/conda-incubator/grayskull/pull/341) to make the original recipe.

Reply:
Great! Seems though like there is a bug in Grayskull as the build script needs to be added like this: https://github.com/conda-forge/sagemaker-inference-toolkit-feedstock/blob/main/recipe/build_base.sh

can you also please open an issue in Grayskull so that this gets incorporated?

Reply:
Will do once this gets merged!